packages/echo/src/types/skip.types.ts

```diff
@@ -1 +1 @@
-export type Skip = (payload: any) => boolean | Promise<boolean>;
+export type Skip = boolean;
```

Comment:
initially, we created the Skip option as a function, however after a small overview with Richard we decided that we could simplify this option for now.
instead of: `skip :() => param.skip`
pass: `skip : param.skip`

I left it as a type so if in the future we will change this behavior it will be easy to change in one place.

let me know what you think.

Reply:
Got it, wrap it in a type so that when we extend, we only need to update the definition and places that need the new behavior, other places would stay the same

Reply:
In the end, we decided to bring back the function type to provide the Product Person the ability to pass inputs to manipulate the skip. 

``` skip :(inputs) => inputs.skip ```